test/.jshintrc

```diff
@@ -16,6 +16,10 @@
 	"globals": {
 		"define": false,
 		"require": false,
+		"module": false,
+		"__dirname": false,
+		"describe": false,
+		"it": false,
```

Comment:
Let's move your just created `test/compiler` into `test-compiler`. That way we isolate the existing tests that use QUnit and your just created ones that use mocha.